Source/WebCore/css/StyleProperties.cpp

```diff
@@ -575,6 +575,8 @@ static std::optional<CSSValueID> fontStretchKeyword(double value)
 
 String StyleProperties::fontValue() const
 {
+    // FIXME: Does this need to be updated to check a lot more things now that CSSPropertyParser::consumeFont resets so much more?
+
```

Comment:
Can we file a bug for it and reference it here? This seems to be needed, the specified style in FF & Chrome when you use a non-expressible property along with the shorthand turns into an empty string for the shorthand.